pkgs/applications/misc/ilspy/default.nix

```diff
@@ -0,0 +1,99 @@
+{ lib
+, stdenv
+, pkgs
+
+, fetchzip
+, unzip
+, autoPatchelfHook
+, makeWrapper
+, makeDesktopItem
+
+#, stdenv.cc.cc.lib
+, lttng-ust
+, libkrb5
+, zlib
+, fontconfig
+
+, openssl_1_1
+, xorg
```

Comment:
```suggestion

, fetchzip
, unzip
, autoPatchelfHook
, makeWrapper
, makeDesktopItem

, lttng-ust
, libkrb5
, zlib
, fontconfig

, openssl_1_1
```

unused

Reply:
Opted to include those for transparency of what is _actually_ used by the package. This is against guidelines?

Reply:
If inputs are unused they are to be removed.